### PR TITLE
msi/hcl.md: update compatible CPU list

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: docs/unified/msi/hcl.md [hunk 1]
```diff
@@ -33,11 +33,17 @@ hardware.
 
     <center>
 
-    === "PRO Z690-A (WIFI) DDR4"
+    === "PRO Z690-A (WIFI) (DDR4)"
         | CPU Model | Dasharo version |  Source  |
         |:---------:|:----------------------:|:---------:|
+        | 12th Gen Intel(R) Core(TM) i9-12900K | v1.1.1 | Dasharo HCL Report |
```
Comment (mkopec) [inline docs/unified/msi/hcl.md:39]: The order seems a bit random, maybe let's sort the list?

Comment (mkopec) [reply docs/unified/msi/hcl.md:39]: the 12900K is at the top hence my comment, the rest of the list looks sorted indeed

Comment (pietrushnic) [reply docs/unified/msi/hcl.md]: @mkopec, now I get your point: 6c6f67fe7bbb

Comment (pietrushnic) [reply docs/unified/msi/hcl.md]: @mkopec ping

**Review by mkopec — APPROVED** [review]

**Pull request merged**